src/comps/forms/PostFormEditModal.vue

```diff
@@ -0,0 +1,93 @@
+<template>
```

Comment:
Talvez usar o componente [CModalCard][1]?

[1]: https://github.com/thecartera/bulma/blob/master/src/ui/CModalCard.vue

Reply:
Tu que sabe, n to ligado. pq usar ele?